### PR TITLE
updates to the docs

### PR DESCRIPTION
**Pull request by JustinSGray**

### CODE REVIEW

File: openmdao/docs/feature_reference/core_features/working_with_derivatives/specifying_partials.rst [hunk 1]
```diff
@@ -16,32 +16,32 @@ should be delcared in the `setup` method of your component.
...
-2. Specifying variables using glob patterns (see https://docs.python.org/3.6/library/fnmatch.html).
+2. Declaring multiple derivatives using glob patterns (see https://docs.python.org/3.6/library/fnmatch.html).
 
 .. embed-code::
     openmdao.jacobians.tests.test_jacobian_features.SimpleCompGlob.setup
 
-3. Using the :code:`val` argument to set a constant partial derivative. Note that if the :code:`val` arugment is used,
-then the partial derivative does not need to be calculated in :code:`compute_partials`.
+3. Using the :code:`val` argument to set a constant partial derivative. Note that this is indtended for cases when the derivative value is constant,
+and hence the derivatives do not ever need to be re computed in :code:`compute_partials`.
```
Comment (naylor-b) [inline openmdao/docs/feature_reference/core_features/working_with_derivatives/specifying_partials.rst]: typo:  re computed

File: openmdao/docs/feature_reference/core_features/working_with_derivatives/specifying_partials.rst [hunk 2]
```diff
@@ -16,32 +16,32 @@ should be delcared in the `setup` method of your component.
...
 
-2. Specifying variables using glob patterns (see https://docs.python.org/3.6/library/fnmatch.html).
+2. Declaring multiple derivatives using glob patterns (see https://docs.python.org/3.6/library/fnmatch.html).
 
 .. embed-code::
     openmdao.jacobians.tests.test_jacobian_features.SimpleCompGlob.setup
 
-3. Using the :code:`val` argument to set a constant partial derivative. Note that if the :code:`val` arugment is used,
-then the partial derivative does not need to be calculated in :code:`compute_partials`.
+3. Using the :code:`val` argument to set a constant partial derivative. Note that this is indtended for cases when the derivative value is constant,
```
Comment (swryan) [inline openmdao/docs/feature_reference/core_features/working_with_derivatives/specifying_partials.rst]: typo: indtended

File: openmdao/docs/feature_reference/visualization_and_analysis/output/index.rst [hunk 3]
```diff
@@ -1,8 +1,14 @@
...
+***********
+
+
 
 .. toctree::
     :maxdepth: 1
 
-    listing_variables.rst
\ No newline at end of file
+    listing_variables.rst
```
Comment (swryan) [inline openmdao/docs/feature_reference/visualization_and_analysis/output/index.rst]: Note that there is an outstanding PR for this doc on which you are a requested reviewer.  That story stated that "the index structure is also weird. Too many levels deep.", which resulted in the `output` directory being removed.  It looks like that is no longer requested?

File: openmdao/docs/feature_reference/visualization_and_analysis/recording/case_reading.rst [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+***************************
+
+A class, `CaseReader`, is provided to read the data from a case recorder file. It will work for any kind of case
+recorder file in OpenMDAO. Currently, OpenMDAO only has a Sqlite case recorder file, but in the future will also have
+an HDF5 case recorder file. `CaseReader` should work for either kind of file as it abstracts away the underlying file
+format.
+
+Here is some simple code showing how to use the `CaseReader` class.
+
+.. code-block:: console
```
Comment (swryan) [inline openmdao/docs/feature_reference/visualization_and_analysis/recording/case_reading.rst]: did we want this test code to be converted to `embed-test` as well? (in another story)

**Pull request merged**